ofl/bevietnampro/METADATA.pb

```diff
@@ -0,0 +1,139 @@
+  name: "Be Vietnam Pro"
+  style: "normal"
+  weight: 400
+  filename: "BeVietnamPro-Regular.ttf"
+  post_script_name: "BeVietnamPro-Regular"
+  full_name: "Be Vietnam Pro Regular"
+  copyright: "Copyright 2021 The Be Vietnam Pro Project Authors (https://github.com/bettergui/BeVietnamPro)"
+}
+fonts {
+  name: "Be Vietnam Pro"
+  style: "italic"
+  weight: 400
+  filename: "BeVietnamPro-Italic.ttf"
+  post_script_name: "BeVietnamPro-Italic"
+  full_name: "Be Vietnam Pro Italic"
+  copyright: "Copyright 2021 The Be Vietnam Pro Project Authors (https://github.com/bettergui/BeVietnamPro)"
+}
+fonts {
+  name: "Be Vietnam Pro"
+  style: "normal"
+  weight: 500
+  filename: "BeVietnamPro-Medium.ttf"
+  post_script_name: "BeVietnamPro-Medium"
+  full_name: "Be Vietnam Pro Medium"
+  copyright: "Copyright 2021 The Be Vietnam Pro Project Authors (https://github.com/bettergui/BeVietnamPro)"
+}
+fonts {
+  name: "Be Vietnam Pro"
+  style: "italic"
+  weight: 500
+  filename: "BeVietnamPro-MediumItalic.ttf"
+  post_script_name: "BeVietnamPro-MediumItalic"
+  full_name: "Be Vietnam Pro Medium Italic"
+  copyright: "Copyright 2021 The Be Vietnam Pro Project Authors (https://github.com/bettergui/BeVietnamPro)"
+}
+fonts {
+  name: "Be Vietnam Pro"
+  style: "normal"
+  weight: 600
+  filename: "BeVietnamPro-SemiBold.ttf"
+  post_script_name: "BeVietnamPro-SemiBold"
+  full_name: "Be Vietnam Pro SemiBold"
+  copyright: "Copyright 2021 The Be Vietnam Pro Project Authors (https://github.com/bettergui/BeVietnamPro)"
+}
+fonts {
+  name: "Be Vietnam Pro"
+  style: "italic"
+  weight: 600
+  filename: "BeVietnamPro-SemiBoldItalic.ttf"
+  post_script_name: "BeVietnamPro-SemiBoldItalic"
+  full_name: "Be Vietnam Pro SemiBold Italic"
+  copyright: "Copyright 2021 The Be Vietnam Pro Project Authors (https://github.com/bettergui/BeVietnamPro)"
+}
+fonts {
+  name: "Be Vietnam Pro"
+  style: "normal"
+  weight: 700
+  filename: "BeVietnamPro-Bold.ttf"
+  post_script_name: "BeVietnamPro-Bold"
+  full_name: "Be Vietnam Pro Bold"
+  copyright: "Copyright 2021 The Be Vietnam Pro Project Authors (https://github.com/bettergui/BeVietnamPro)"
+}
+fonts {
+  name: "Be Vietnam Pro"
+  style: "italic"
+  weight: 700
+  filename: "BeVietnamPro-BoldItalic.ttf"
+  post_script_name: "BeVietnamPro-BoldItalic"
+  full_name: "Be Vietnam Pro Bold Italic"
+  copyright: "Copyright 2021 The Be Vietnam Pro Project Authors (https://github.com/bettergui/BeVietnamPro)"
+}
+fonts {
+  name: "Be Vietnam Pro"
+  style: "normal"
+  weight: 900
+  filename: "BeVietnamPro-ExtraBold.ttf"
+  post_script_name: "BeVietnamPro-ExtraBold"
+  full_name: "Be Vietnam Pro ExtraBold"
+  copyright: "Copyright 2021 The Be Vietnam Pro Project Authors (https://github.com/bettergui/BeVietnamPro)"
+}
+fonts {
+  name: "Be Vietnam Pro"
+  style: "italic"
+  weight: 900
+  filename: "BeVietnamPro-ExtraBoldItalic.ttf"
+  post_script_name: "BeVietnamPro-ExtraBoldItalic"
+  full_name: "Be Vietnam Pro ExtraBold Italic"
```

Comment:
Your font weights are wrong. ExtraBold should be 800, not 900. Please refer to this table, https://github.com/googlefonts/gf-docs/tree/main/Spec#supported-styles